com.unity.render-pipelines.high-definition/Runtime/Material/Lit/Lit.hlsl

```diff
@@ -96,40 +96,40 @@ static const uint kFeatureVariantFlags[NUM_FEATURE_VARIANTS] =
     // Precomputed illumination (no dynamic lights) with standard, SSS and transmission
     /*  1 */ LIGHTFEATUREFLAGS_SKY | LIGHTFEATUREFLAGS_ENV | LIGHTFEATUREFLAGS_SSREFLECTION | MATERIALFEATUREFLAGS_LIT_SUBSURFACE_SCATTERING | MATERIALFEATUREFLAGS_LIT_TRANSMISSION | MATERIALFEATUREFLAGS_LIT_STANDARD,
     // Precomputed illumination (no dynamic lights) for all material types
-    /*  2 */ LIGHTFEATUREFLAGS_SKY | LIGHTFEATUREFLAGS_ENV | LIGHTFEATUREFLAGS_PROBE_VOLUME | LIGHTFEATUREFLAGS_SSREFLECTION | MATERIAL_FEATURE_MASK_FLAGS,
```

Comment:
@FrancescoC-unity would like to chat with you about this change. My understanding is that LIGHTFEATUREFLAGS_PROBE_VOLUME was added for V1. Which mean that with V2 this code is currently inefficient and force to always used the worst case variant. So this change here should currently bring back performance. Let's chat to validate it is true

Reply:
Let's chat about it yes, the thing was indeed added for PV1